### ISSUE TITLE
Go in linguist

### ISSUE THREAD

**Issue by ShalokShalom** [github/linguist#3952]
Linguist seems to avoid .go files in your own repo:

https://github.com/github/linguist
https://github.com/github/linguist/tree/master/tools/grammars/compiler

**Comment by lildude** [comment]
This is because Linguist is a victim of its own success 😂 

The version of Linguist currently used on GitHub.com does not include the changes from https://github.com/github/linguist/pull/3919 which means everything under `tools/` don't count toward the stats. 

We're planning a new release early next year which will "fix" this.

**Comment by ShalokShalom** [comment]
And what stops this merge?

**Comment by lildude** [comment]
The PR has been merged already. We've not released a new version of Linguist yet as I'm waiting on https://github.com/github/linguist/pull/3807 that I'd like to include in the next release as it's a significant change that will go well with the other significant changes & improvements we've made recently. 

More importantly, we're entering the Christmas period in which we have a "social freeze" on deployments to GitHub.com in which we discourage deploying anything other than urgent bug fixes to reduce the chances of introducing a problem that could pull peeps away from their families.

A new release will be made early next year and deployed to GitHub.com on the same day.

If you're affected by `tools/` being marked as vendored in any of your own repos, a [manual override](https://github.com/github/linguist#vendored-code) can be used in the mean time.
